akka-docs/src/main/paradox/additional/rolling-updates.md

```diff
@@ -0,0 +1,90 @@
+   during the update old nodes must be able to read data stored by new nodes.
+
+There are many more application specific aspects for serialization changes during rolling upgrades to consider. 
+For example, whether to allow dropped messages or tear down the TCP connection when the manifest is unknown.
+
+* When some message loss during a rolling upgrade is acceptable versus a full shutdown and restart, assuming the application recovers afterwards 
+    - If a `java.io.NotSerializableException` is thrown in `fromBinary` this is treated as a transient problem, the issue logged and the message is dropped
+    - If other exceptions are thrown it can be an indication of corrupt bytes from the underlying transport, and the connection is broken
+* For more zero-impact rolling upgrades, it is important to consider a strategy for serialization format that can be evolved. You can find advice in
+@ref:[Persistence - Schema Evolution](../persistence-schema-evolution.md), which also applies to
+remote messages when deploying with rolling updates.
+
+One approach to retiring a serializer without downtime is carried out in @ref:[two rolling upgrade steps to switch to the new serializer](../serialization.md#rolling-upgrades). 
+
+## Cluster Sharding
+
+During a rolling upgrade, sharded entities receiving traffic may be moved during @ref:[shard rebalancing](../cluster-sharding.md#shard-rebalancing), 
+to an old or new node in the cluster, based on the pluggable allocation strategy and settings.
+When an old node is stopped the shards that were running on it may be allocated to one of the
+other old nodes remaining in the cluster. See @ref[ClusterSingleton](#cluster-singleton) for a useful `ShardCoordinator` optimization.
+
+There are some cases when @ref:[a full cluster restart is needed](../cluster-sharding.md#rolling-upgrades).
+
+## Cluster Singleton
+
+It's more efficient to avoid moving a `ClusterSingleton` more than necessary because they typically have to recover their state
+and it might introduce unnecessary delays.
+
+An optional optimization is to leave the oldest node running a `ClusterSingleton` until last
+to avoid it having to move more than once. 
+
+## Migrating Untyped to Typed
+
+It is recommended with a two step approach:
+
+* Deploy with the new nodes set to `akka.cluster.configuration-compatibility-check.enforce-on-join = off`
+and ensure all nodes are in this state
+* Deploy again and with the new nodes set to `akka.cluster.configuration-compatibility-check.enforce-on-join = on`. 
+
+The configuration from existing nodes should pass the @ref:[Cluster Configuration Compatibility Checks](#cluster-configuration-compatibility-check).
+Find out more about coexisting and @ref:[untyped to typed](../typed/coexisting.md#untyped-to-typed). 
+
+### With Cluster Sharding and Persistence
+
+Rolling upgrades where shards on old nodes are running untyped persistent actors 
```

Comment:
What does that mean exactly? Is that about migrating persistent entities from classic to typed? Or that on new nodes you could have new entities written with typed style?

Reply:
Removing it for adding when samples are done, but this is something user verified https://github.com/akka/akka/issues/26718